tests/ZendTest/Code/Reflection/TestAsset/TestSampleClass13.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace ZendTest\Code\Reflection\TestAsset;
+
+/** Short Class Description */
+class TestSampleClass13
+{
+    /** Short Method Description */
+    public function doSomething()
+    {}
```

Comment:
This asset is actually supposed to have this particular structure to trigger a failure